mypy.ini

```diff
@@ -30,7 +30,6 @@ pretty = True
 show_column_numbers = True
 show_error_codes = True
 show_error_context = True
-show_none_errors = True
```

Comment:
This disables the check globally. Can you disable the check locally in a single file instead, if you need to?

Reply:
Oh, I don't need to. I saw this error in my local CI. Maybe it was the wrong to do. 

`mypy.ini: [mypy]: Unrecognized option: show_none_errors = True`

Reply:
Ah, gotcha now. You're right: it's good to remove this.